float32 ch1_value
float32 ch2_value
float32 ch3_value
float32 ch4_value
float32 ch5_value
float32 ch6_value
float32 ch7_value
float32 ch8_value
